src/validation.ts

```diff
@@ -81,6 +83,12 @@ export function validateOptions(options: any): never | void {
   })
 
   validateType({ name: 'name', value: name, type: 'string', optional: true })
+  validateType({
```

Comment:
@aaronbarnardsound does `appName` need validation as well?

Reply:
`appName` is what is sent to the server, but when initializing the SDK it is `name` so is already validated. I decided to use the `name` parameter which was already there to identify Onboard/Notify, but was just never sent to the server. Figured using `appName` and `appVersion` would be more clear when sending to the server to distinguish from the SDK `version` that is already sent.